support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -255,6 +256,11 @@ export function Checkout({ geoId, appConfig, abParticipations }: Props) {
 		);
 	}, []);
 
+	const checkoutSessionId = window.location.hash;
+	const persistedFormData = checkoutSessionId
```

Comment:
If there's a value in the hash param and we don't manage to retrieve persisted form data (because if fails to parse of it doesn't match the checkout session ID) should we update `window.location` to remove the hash param?

Reply:
I've made this change.